daemon/daemon.go

```diff
@@ -434,7 +434,7 @@ var (
 	rebootNoticeWait       = 3 * time.Second
 	rebootWaitTimeout      = 10 * time.Minute
 	rebootRetryWaitTimeout = 5 * time.Minute
-	rebootMaxTentatives    = 3
+	rebootMaxTentative     = 3
```

Comment:
I think using "tentative" here is still wrong, "attempt" is closer to the intention
```suggestion
	rebootMaxAttempts     = 3
```

Reply:
sounds like a good idea